30 May 2022 22:41:52,418 ERROR   WARNING: This chart is deprecated
30 May 2022 22:41:52,418 ERROR   Error: execution error at (distribution/templates/_helpers.tpl:88:22): A valid .Values.redis.redisPassword entry required!
30 May 2022 22:41:52,419 ERROR   
30 May 2022 22:41:52,419 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 22:41:52,419 ERROR   Error running template command. Exit Value = 1.
30 May 2022 22:41:52,419 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template distribution 
30 May 2022 22:41:52,419 ERROR   Exception rendering template for distribution:0.4.3 : Error running template command. Exit Value = 1.
30 May 2022 22:41:52,419 ERROR   Error resolving chart dependencies: Exception rendering template for distribution:0.4.3 : Error running template command. Exit Value = 1.
30 May 2022 22:41:56,559 ERROR   Error resolving chart dependencies: Exception updating chart repo in directory /tmp/com.melahn.util.helm.ChartMap.Temporary.3259370546947319997/distribution.  Exit code: 1.  Possibly you cannot access one of your remote charts repos.
30 May 2022 22:41:56,561 ERROR   Chart distribution:0.4.3 failed to print even with the refresh option.
30 May 2022 22:41:58,163 ERROR   WARNING: This chart is deprecated
30 May 2022 22:41:58,163 ERROR   Error: execution error at (distribution/templates/_helpers.tpl:88:22): A valid .Values.redis.redisPassword entry required!
30 May 2022 22:41:58,163 ERROR   
30 May 2022 22:41:58,163 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 22:41:58,163 ERROR   Error running template command. Exit Value = 1.
30 May 2022 22:41:58,163 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template distribution 
30 May 2022 22:41:58,164 ERROR   Exception rendering template for distribution:0.4.2 : Error running template command. Exit Value = 1.
30 May 2022 22:41:58,164 ERROR   Error resolving chart dependencies: Exception rendering template for distribution:0.4.2 : Error running template command. Exit Value = 1.
30 May 2022 22:42:02,281 ERROR   Error resolving chart dependencies: Exception updating chart repo in directory /tmp/com.melahn.util.helm.ChartMap.Temporary.5205452090963597879/distribution.  Exit code: 1.  Possibly you cannot access one of your remote charts repos.
30 May 2022 22:42:02,281 ERROR   Chart distribution:0.4.2 failed to print even with the refresh option.
30 May 2022 22:42:03,882 ERROR   Error: execution error at (distribution/templates/_helpers.tpl:88:22): A valid .Values.redis.redisPassword entry required!
30 May 2022 22:42:03,882 ERROR   
30 May 2022 22:42:03,882 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 22:42:03,882 ERROR   Error running template command. Exit Value = 1.
30 May 2022 22:42:03,882 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template distribution 
30 May 2022 22:42:03,882 ERROR   Exception rendering template for distribution:0.4.1 : Error running template command. Exit Value = 1.
30 May 2022 22:42:03,882 ERROR   Error resolving chart dependencies: Exception rendering template for distribution:0.4.1 : Error running template command. Exit Value = 1.
30 May 2022 22:42:07,999 ERROR   Error resolving chart dependencies: Exception updating chart repo in directory /tmp/com.melahn.util.helm.ChartMap.Temporary.206731158534223466/distribution.  Exit code: 1.  Possibly you cannot access one of your remote charts repos.
30 May 2022 22:42:07,999 ERROR   Chart distribution:0.4.1 failed to print even with the refresh option.
30 May 2022 23:02:35,499 ERROR   Error: YAML parse error on kuberhealthy/templates/deployment.yaml: error converting YAML to JSON: yaml: line 41: mapping values are not allowed in this context
30 May 2022 23:02:35,499 ERROR   
30 May 2022 23:02:35,499 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:02:35,499 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:02:35,500 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template kuberhealthy 
30 May 2022 23:02:35,500 ERROR   Exception rendering template for kuberhealthy:1.2.5 : Error running template command. Exit Value = 1.
30 May 2022 23:02:35,500 ERROR   Error resolving chart dependencies: Exception rendering template for kuberhealthy:1.2.5 : Error running template command. Exit Value = 1.
30 May 2022 23:02:37,015 ERROR   Error: YAML parse error on kuberhealthy/templates/deployment.yaml: error converting YAML to JSON: yaml: line 41: mapping values are not allowed in this context
30 May 2022 23:02:37,015 ERROR   
30 May 2022 23:02:37,015 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:02:37,015 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:02:37,015 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template kuberhealthy 
30 May 2022 23:02:37,016 ERROR   Exception rendering template for kuberhealthy:1.2.5 : Error running template command. Exit Value = 1.
30 May 2022 23:02:37,016 ERROR   Error resolving chart dependencies: Exception rendering template for kuberhealthy:1.2.5 : Error running template command. Exit Value = 1.
30 May 2022 23:02:37,016 ERROR   Chart kuberhealthy:1.2.5 failed to print even with the refresh option.
30 May 2022 23:08:42,958 ERROR   WARNING: This chart is deprecated
30 May 2022 23:08:42,958 ERROR   Error: execution error at (mission-control/templates/mongodb-secret.yaml:12:21): A valid .Values.mongodb.db.adminPassword entry required!
30 May 2022 23:08:42,958 ERROR   
30 May 2022 23:08:42,958 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:08:42,958 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:08:42,959 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template mission-control 
30 May 2022 23:08:42,959 ERROR   Exception rendering template for mission-control:0.4.4 : Error running template command. Exit Value = 1.
30 May 2022 23:08:42,959 ERROR   Error resolving chart dependencies: Exception rendering template for mission-control:0.4.4 : Error running template command. Exit Value = 1.
30 May 2022 23:08:47,093 ERROR   Error resolving chart dependencies: Exception updating chart repo in directory /tmp/com.melahn.util.helm.ChartMap.Temporary.4305621661100338562/mission-control.  Exit code: 1.  Possibly you cannot access one of your remote charts repos.
30 May 2022 23:08:47,093 ERROR   Chart mission-control:0.4.4 failed to print even with the refresh option.
30 May 2022 23:08:48,785 ERROR   WARNING: This chart is deprecated
30 May 2022 23:08:48,785 ERROR   Error: execution error at (mission-control/templates/mongodb-secret.yaml:12:21): A valid .Values.mongodb.db.adminPassword entry required!
30 May 2022 23:08:48,785 ERROR   
30 May 2022 23:08:48,785 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:08:48,785 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:08:48,785 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template mission-control 
30 May 2022 23:08:48,785 ERROR   Exception rendering template for mission-control:0.4.3 : Error running template command. Exit Value = 1.
30 May 2022 23:08:48,785 ERROR   Error resolving chart dependencies: Exception rendering template for mission-control:0.4.3 : Error running template command. Exit Value = 1.
30 May 2022 23:08:52,870 ERROR   Error resolving chart dependencies: Exception updating chart repo in directory /tmp/com.melahn.util.helm.ChartMap.Temporary.10575944997824325031/mission-control.  Exit code: 1.  Possibly you cannot access one of your remote charts repos.
30 May 2022 23:08:52,871 ERROR   Chart mission-control:0.4.3 failed to print even with the refresh option.
30 May 2022 23:08:54,482 ERROR   Error: execution error at (mission-control/templates/mongodb-secret.yaml:12:21): A valid .Values.mongodb.db.adminPassword entry required!
30 May 2022 23:08:54,483 ERROR   
30 May 2022 23:08:54,483 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:08:54,483 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:08:54,483 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template mission-control 
30 May 2022 23:08:54,483 ERROR   Exception rendering template for mission-control:0.4.2 : Error running template command. Exit Value = 1.
30 May 2022 23:08:54,483 ERROR   Error resolving chart dependencies: Exception rendering template for mission-control:0.4.2 : Error running template command. Exit Value = 1.
30 May 2022 23:08:58,647 ERROR   Error resolving chart dependencies: Exception updating chart repo in directory /tmp/com.melahn.util.helm.ChartMap.Temporary.718799195658040983/mission-control.  Exit code: 1.  Possibly you cannot access one of your remote charts repos.
30 May 2022 23:08:58,647 ERROR   Chart mission-control:0.4.2 failed to print even with the refresh option.
30 May 2022 23:09:24,023 ERROR   Error: YAML parse error on mongodb-replicaset/templates/mongodb-statefulset.yaml: error converting YAML to JSON: yaml: line 123: mapping values are not allowed in this context
30 May 2022 23:09:24,023 ERROR   
30 May 2022 23:09:24,023 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:09:24,023 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:09:24,023 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template mongodb-replicaset 
30 May 2022 23:09:24,023 ERROR   Exception rendering template for mongodb-replicaset:3.17.0 : Error running template command. Exit Value = 1.
30 May 2022 23:09:24,023 ERROR   Error resolving chart dependencies: Exception rendering template for mongodb-replicaset:3.17.0 : Error running template command. Exit Value = 1.
30 May 2022 23:09:25,585 ERROR   Error: YAML parse error on mongodb-replicaset/templates/mongodb-statefulset.yaml: error converting YAML to JSON: yaml: line 123: mapping values are not allowed in this context
30 May 2022 23:09:25,585 ERROR   
30 May 2022 23:09:25,585 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:09:25,585 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:09:25,585 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template mongodb-replicaset 
30 May 2022 23:09:25,585 ERROR   Exception rendering template for mongodb-replicaset:3.17.0 : Error running template command. Exit Value = 1.
30 May 2022 23:09:25,585 ERROR   Error resolving chart dependencies: Exception rendering template for mongodb-replicaset:3.17.0 : Error running template command. Exit Value = 1.
30 May 2022 23:09:25,586 ERROR   Chart mongodb-replicaset:3.17.0 failed to print even with the refresh option.
30 May 2022 23:10:32,911 ERROR   Error: template: mysqldump/templates/configmap.yaml:49:10: executing "mysqldump/templates/configmap.yaml" at <(.Values.persistence.enabled) or (.Values.persistentVolumeClaim)>: can't give argument to non-function .Values.persistence.enabled
30 May 2022 23:10:32,911 ERROR   
30 May 2022 23:10:32,911 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:10:32,911 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:10:32,911 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template mysqldump 
30 May 2022 23:10:32,911 ERROR   Exception rendering template for mysqldump:2.6.0 : Error running template command. Exit Value = 1.
30 May 2022 23:10:32,911 ERROR   Error resolving chart dependencies: Exception rendering template for mysqldump:2.6.0 : Error running template command. Exit Value = 1.
30 May 2022 23:10:34,444 ERROR   Error: template: mysqldump/templates/configmap.yaml:49:10: executing "mysqldump/templates/configmap.yaml" at <(.Values.persistence.enabled) or (.Values.persistentVolumeClaim)>: can't give argument to non-function .Values.persistence.enabled
30 May 2022 23:10:34,444 ERROR   
30 May 2022 23:10:34,444 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:10:34,444 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:10:34,444 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template mysqldump 
30 May 2022 23:10:34,444 ERROR   Exception rendering template for mysqldump:2.6.0 : Error running template command. Exit Value = 1.
30 May 2022 23:10:34,444 ERROR   Error resolving chart dependencies: Exception rendering template for mysqldump:2.6.0 : Error running template command. Exit Value = 1.
30 May 2022 23:10:34,444 ERROR   Chart mysqldump:2.6.0 failed to print even with the refresh option.
30 May 2022 23:17:03,460 ERROR   Exception rendering template for pgadmin:1.2.3 : Can't construct a java object for tag:yaml.org,2002:string; exception=Class not found: string
 in 'reader', line 118, column 22:
                  value: !!string False
                         ^

30 May 2022 23:17:03,460 ERROR   Error resolving chart dependencies: Exception rendering template for pgadmin:1.2.3 : Can't construct a java object for tag:yaml.org,2002:string; exception=Class not found: string
 in 'reader', line 118, column 22:
                  value: !!string False
                         ^

30 May 2022 23:17:04,976 ERROR   Exception rendering template for pgadmin:1.2.3 : Can't construct a java object for tag:yaml.org,2002:string; exception=Class not found: string
 in 'reader', line 118, column 22:
                  value: !!string False
                         ^

30 May 2022 23:17:04,976 ERROR   Error resolving chart dependencies: Exception rendering template for pgadmin:1.2.3 : Can't construct a java object for tag:yaml.org,2002:string; exception=Class not found: string
 in 'reader', line 118, column 22:
                  value: !!string False
                         ^

30 May 2022 23:17:04,976 ERROR   Chart pgadmin:1.2.3 failed to print even with the refresh option.
30 May 2022 23:17:06,689 ERROR   Exception rendering template for pgadmin:1.2.2 : Can't construct a java object for tag:yaml.org,2002:string; exception=Class not found: string
 in 'reader', line 118, column 22:
                  value: !!string False
                         ^

30 May 2022 23:17:06,689 ERROR   Error resolving chart dependencies: Exception rendering template for pgadmin:1.2.2 : Can't construct a java object for tag:yaml.org,2002:string; exception=Class not found: string
 in 'reader', line 118, column 22:
                  value: !!string False
                         ^

30 May 2022 23:17:08,170 ERROR   Exception rendering template for pgadmin:1.2.2 : Can't construct a java object for tag:yaml.org,2002:string; exception=Class not found: string
 in 'reader', line 118, column 22:
                  value: !!string False
                         ^

30 May 2022 23:17:08,170 ERROR   Error resolving chart dependencies: Exception rendering template for pgadmin:1.2.2 : Can't construct a java object for tag:yaml.org,2002:string; exception=Class not found: string
 in 'reader', line 118, column 22:
                  value: !!string False
                         ^

30 May 2022 23:17:08,170 ERROR   Chart pgadmin:1.2.2 failed to print even with the refresh option.
30 May 2022 23:20:38,715 ERROR   WARNING: This chart is deprecated
30 May 2022 23:20:38,715 ERROR   Error: execution error at (prometheus-mongodb-exporter/templates/secret.yaml:13:40): A MongoDB URI is required
30 May 2022 23:20:38,715 ERROR   
30 May 2022 23:20:38,715 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:20:38,715 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:20:38,715 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template prometheus-mongodb-exporter 
30 May 2022 23:20:38,715 ERROR   Exception rendering template for prometheus-mongodb-exporter:2.8.1 : Error running template command. Exit Value = 1.
30 May 2022 23:20:38,715 ERROR   Error resolving chart dependencies: Exception rendering template for prometheus-mongodb-exporter:2.8.1 : Error running template command. Exit Value = 1.
30 May 2022 23:20:40,206 ERROR   WARNING: This chart is deprecated
30 May 2022 23:20:40,206 ERROR   Error: execution error at (prometheus-mongodb-exporter/templates/secret.yaml:13:40): A MongoDB URI is required
30 May 2022 23:20:40,206 ERROR   
30 May 2022 23:20:40,206 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:20:40,206 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:20:40,206 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template prometheus-mongodb-exporter 
30 May 2022 23:20:40,206 ERROR   Exception rendering template for prometheus-mongodb-exporter:2.8.1 : Error running template command. Exit Value = 1.
30 May 2022 23:20:40,206 ERROR   Error resolving chart dependencies: Exception rendering template for prometheus-mongodb-exporter:2.8.1 : Error running template command. Exit Value = 1.
30 May 2022 23:20:40,206 ERROR   Chart prometheus-mongodb-exporter:2.8.1 failed to print even with the refresh option.
30 May 2022 23:20:41,786 ERROR   Error: execution error at (prometheus-mongodb-exporter/templates/secret.yaml:13:40): A MongoDB URI is required
30 May 2022 23:20:41,786 ERROR   
30 May 2022 23:20:41,786 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:20:41,786 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:20:41,786 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template prometheus-mongodb-exporter 
30 May 2022 23:20:41,786 ERROR   Exception rendering template for prometheus-mongodb-exporter:2.8.0 : Error running template command. Exit Value = 1.
30 May 2022 23:20:41,786 ERROR   Error resolving chart dependencies: Exception rendering template for prometheus-mongodb-exporter:2.8.0 : Error running template command. Exit Value = 1.
30 May 2022 23:20:43,272 ERROR   Error: execution error at (prometheus-mongodb-exporter/templates/secret.yaml:13:40): A MongoDB URI is required
30 May 2022 23:20:43,272 ERROR   
30 May 2022 23:20:43,272 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:20:43,272 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:20:43,272 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template prometheus-mongodb-exporter 
30 May 2022 23:20:43,272 ERROR   Exception rendering template for prometheus-mongodb-exporter:2.8.0 : Error running template command. Exit Value = 1.
30 May 2022 23:20:43,272 ERROR   Error resolving chart dependencies: Exception rendering template for prometheus-mongodb-exporter:2.8.0 : Error running template command. Exit Value = 1.
30 May 2022 23:20:43,272 ERROR   Chart prometheus-mongodb-exporter:2.8.0 failed to print even with the refresh option.
30 May 2022 23:20:44,860 ERROR   Error: execution error at (prometheus-mongodb-exporter/templates/secret.yaml:13:40): A MongoDB URI is required
30 May 2022 23:20:44,860 ERROR   
30 May 2022 23:20:44,860 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:20:44,860 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:20:44,860 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template prometheus-mongodb-exporter 
30 May 2022 23:20:44,860 ERROR   Exception rendering template for prometheus-mongodb-exporter:2.7.0 : Error running template command. Exit Value = 1.
30 May 2022 23:20:44,860 ERROR   Error resolving chart dependencies: Exception rendering template for prometheus-mongodb-exporter:2.7.0 : Error running template command. Exit Value = 1.
30 May 2022 23:20:46,352 ERROR   Error: execution error at (prometheus-mongodb-exporter/templates/secret.yaml:13:40): A MongoDB URI is required
30 May 2022 23:20:46,352 ERROR   
30 May 2022 23:20:46,352 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:20:46,352 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:20:46,352 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template prometheus-mongodb-exporter 
30 May 2022 23:20:46,352 ERROR   Exception rendering template for prometheus-mongodb-exporter:2.7.0 : Error running template command. Exit Value = 1.
30 May 2022 23:20:46,352 ERROR   Error resolving chart dependencies: Exception rendering template for prometheus-mongodb-exporter:2.7.0 : Error running template command. Exit Value = 1.
30 May 2022 23:20:46,352 ERROR   Chart prometheus-mongodb-exporter:2.7.0 failed to print even with the refresh option.
30 May 2022 23:25:25,276 ERROR   WARNING: This chart is deprecated
30 May 2022 23:25:25,276 ERROR   Error: execution error at (rocketchat/templates/secret.yaml:21:19): Please specify a MongoDB password
30 May 2022 23:25:25,276 ERROR   
30 May 2022 23:25:25,276 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:25:25,276 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:25:25,276 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template rocketchat 
30 May 2022 23:25:25,276 ERROR   Exception rendering template for rocketchat:2.0.10 : Error running template command. Exit Value = 1.
30 May 2022 23:25:25,276 ERROR   Error resolving chart dependencies: Exception rendering template for rocketchat:2.0.10 : Error running template command. Exit Value = 1.
30 May 2022 23:25:35,330 ERROR   WARNING: This chart is deprecated
30 May 2022 23:25:35,330 ERROR   Error: execution error at (rocketchat/templates/secret.yaml:21:19): Please specify a MongoDB password
30 May 2022 23:25:35,330 ERROR   
30 May 2022 23:25:35,330 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:25:35,330 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:25:35,330 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template rocketchat 
30 May 2022 23:25:35,330 ERROR   Exception rendering template for rocketchat:2.0.10 : Error running template command. Exit Value = 1.
30 May 2022 23:25:35,330 ERROR   Error resolving chart dependencies: Exception rendering template for rocketchat:2.0.10 : Error running template command. Exit Value = 1.
30 May 2022 23:25:35,331 ERROR   Chart rocketchat:2.0.10 failed to print even with the refresh option.
30 May 2022 23:25:36,996 ERROR   Error: execution error at (rocketchat/templates/secret.yaml:21:19): Please specify a MongoDB password
30 May 2022 23:25:36,996 ERROR   
30 May 2022 23:25:36,996 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:25:36,996 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:25:36,996 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template rocketchat 
30 May 2022 23:25:36,996 ERROR   Exception rendering template for rocketchat:2.0.9 : Error running template command. Exit Value = 1.
30 May 2022 23:25:36,996 ERROR   Error resolving chart dependencies: Exception rendering template for rocketchat:2.0.9 : Error running template command. Exit Value = 1.
30 May 2022 23:25:46,842 ERROR   Error: execution error at (rocketchat/templates/secret.yaml:21:19): Please specify a MongoDB password
30 May 2022 23:25:46,842 ERROR   
30 May 2022 23:25:46,842 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:25:46,842 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:25:46,842 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template rocketchat 
30 May 2022 23:25:46,842 ERROR   Exception rendering template for rocketchat:2.0.9 : Error running template command. Exit Value = 1.
30 May 2022 23:25:46,842 ERROR   Error resolving chart dependencies: Exception rendering template for rocketchat:2.0.9 : Error running template command. Exit Value = 1.
30 May 2022 23:25:46,843 ERROR   Chart rocketchat:2.0.9 failed to print even with the refresh option.
30 May 2022 23:25:48,505 ERROR   Error: execution error at (rocketchat/templates/secret.yaml:21:19): Please specify a MongoDB password
30 May 2022 23:25:48,505 ERROR   
30 May 2022 23:25:48,505 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:25:48,505 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:25:48,505 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template rocketchat 
30 May 2022 23:25:48,505 ERROR   Exception rendering template for rocketchat:2.0.7 : Error running template command. Exit Value = 1.
30 May 2022 23:25:48,505 ERROR   Error resolving chart dependencies: Exception rendering template for rocketchat:2.0.7 : Error running template command. Exit Value = 1.
30 May 2022 23:25:52,745 ERROR   Error resolving chart dependencies: Exception updating chart repo in directory /tmp/com.melahn.util.helm.ChartMap.Temporary.15435182030254232229/rocketchat.  Exit code: 1.  Possibly you cannot access one of your remote charts repos.
30 May 2022 23:25:52,745 ERROR   Chart rocketchat:2.0.7 failed to print even with the refresh option.
30 May 2022 23:26:25,461 ERROR   Error in net.sourceforge.plantuml.GeneratedImage trying to generate image from /home/runner/work/helm-chartmap-generator/helm-chartmap-generator/docs/kubernetes-charts.storage.googleapis.com//home/runner/work/helm-chartmap-generator/helm-chartmap-generator/docs/kubernetes-charts.storage.googleapis.com/satisfy-1.1.2.puml
30 May 2022 23:26:30,358 ERROR   Error in net.sourceforge.plantuml.GeneratedImage trying to generate image from /home/runner/work/helm-chartmap-generator/helm-chartmap-generator/docs/kubernetes-charts.storage.googleapis.com//home/runner/work/helm-chartmap-generator/helm-chartmap-generator/docs/kubernetes-charts.storage.googleapis.com/satisfy-1.1.1.puml
30 May 2022 23:26:35,181 ERROR   Error in net.sourceforge.plantuml.GeneratedImage trying to generate image from /home/runner/work/helm-chartmap-generator/helm-chartmap-generator/docs/kubernetes-charts.storage.googleapis.com//home/runner/work/helm-chartmap-generator/helm-chartmap-generator/docs/kubernetes-charts.storage.googleapis.com/satisfy-1.1.0.puml
30 May 2022 23:28:57,974 ERROR   WARNING: This chart is deprecated
30 May 2022 23:28:57,974 ERROR   Error: template: signalsciences/templates/NOTES.txt:1:12: executing "signalsciences/templates/NOTES.txt" at <(or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)) (or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.secretAccessKey))>: can't give argument to non-function or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)
30 May 2022 23:28:57,974 ERROR   
30 May 2022 23:28:57,974 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:28:57,974 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:28:57,974 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template signalsciences 
30 May 2022 23:28:57,975 ERROR   Exception rendering template for signalsciences:2.0.2 : Error running template command. Exit Value = 1.
30 May 2022 23:28:57,975 ERROR   Error resolving chart dependencies: Exception rendering template for signalsciences:2.0.2 : Error running template command. Exit Value = 1.
30 May 2022 23:28:59,484 ERROR   WARNING: This chart is deprecated
30 May 2022 23:28:59,484 ERROR   Error: template: signalsciences/templates/NOTES.txt:1:12: executing "signalsciences/templates/NOTES.txt" at <(or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)) (or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.secretAccessKey))>: can't give argument to non-function or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)
30 May 2022 23:28:59,484 ERROR   
30 May 2022 23:28:59,484 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:28:59,484 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:28:59,484 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template signalsciences 
30 May 2022 23:28:59,484 ERROR   Exception rendering template for signalsciences:2.0.2 : Error running template command. Exit Value = 1.
30 May 2022 23:28:59,484 ERROR   Error resolving chart dependencies: Exception rendering template for signalsciences:2.0.2 : Error running template command. Exit Value = 1.
30 May 2022 23:28:59,484 ERROR   Chart signalsciences:2.0.2 failed to print even with the refresh option.
30 May 2022 23:29:01,034 ERROR   Error: template: signalsciences/templates/NOTES.txt:1:12: executing "signalsciences/templates/NOTES.txt" at <(or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)) (or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.secretAccessKey))>: can't give argument to non-function or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)
30 May 2022 23:29:01,034 ERROR   
30 May 2022 23:29:01,034 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:29:01,034 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:29:01,034 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template signalsciences 
30 May 2022 23:29:01,034 ERROR   Exception rendering template for signalsciences:2.0.1 : Error running template command. Exit Value = 1.
30 May 2022 23:29:01,034 ERROR   Error resolving chart dependencies: Exception rendering template for signalsciences:2.0.1 : Error running template command. Exit Value = 1.
30 May 2022 23:29:02,549 ERROR   Error: template: signalsciences/templates/NOTES.txt:1:12: executing "signalsciences/templates/NOTES.txt" at <(or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)) (or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.secretAccessKey))>: can't give argument to non-function or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)
30 May 2022 23:29:02,549 ERROR   
30 May 2022 23:29:02,549 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:29:02,549 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:29:02,549 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template signalsciences 
30 May 2022 23:29:02,549 ERROR   Exception rendering template for signalsciences:2.0.1 : Error running template command. Exit Value = 1.
30 May 2022 23:29:02,549 ERROR   Error resolving chart dependencies: Exception rendering template for signalsciences:2.0.1 : Error running template command. Exit Value = 1.
30 May 2022 23:29:02,549 ERROR   Chart signalsciences:2.0.1 failed to print even with the refresh option.
30 May 2022 23:29:04,079 ERROR   Error: template: signalsciences/templates/NOTES.txt:1:12: executing "signalsciences/templates/NOTES.txt" at <(or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)) (or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.secretAccessKey))>: can't give argument to non-function or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)
30 May 2022 23:29:04,079 ERROR   
30 May 2022 23:29:04,079 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:29:04,079 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:29:04,079 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template signalsciences 
30 May 2022 23:29:04,079 ERROR   Exception rendering template for signalsciences:2.0.0 : Error running template command. Exit Value = 1.
30 May 2022 23:29:04,079 ERROR   Error resolving chart dependencies: Exception rendering template for signalsciences:2.0.0 : Error running template command. Exit Value = 1.
30 May 2022 23:29:05,605 ERROR   Error: template: signalsciences/templates/NOTES.txt:1:12: executing "signalsciences/templates/NOTES.txt" at <(or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)) (or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.secretAccessKey))>: can't give argument to non-function or (.Values.signalsciences.accessKeyIdExistingSecret) (.Values.signalsciences.accessKeyId)
30 May 2022 23:29:05,605 ERROR   
30 May 2022 23:29:05,605 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:29:05,605 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:29:05,605 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template signalsciences 
30 May 2022 23:29:05,605 ERROR   Exception rendering template for signalsciences:2.0.0 : Error running template command. Exit Value = 1.
30 May 2022 23:29:05,605 ERROR   Error resolving chart dependencies: Exception rendering template for signalsciences:2.0.0 : Error running template command. Exit Value = 1.
30 May 2022 23:29:05,605 ERROR   Chart signalsciences:2.0.0 failed to print even with the refresh option.
30 May 2022 23:30:32,098 ERROR   WARNING: This chart is deprecated
30 May 2022 23:30:32,098 ERROR   Error: template: spinnaker/charts/minio/templates/deployment.yaml:210:20: executing "spinnaker/charts/minio/templates/deployment.yaml" at <(not .Values.gcsgateway.enabled) (not .Values.azuregateway.enabled) (not .Values.s3gateway.enabled) (not .Values.b2gateway.enabled)>: can't give argument to non-function not .Values.gcsgateway.enabled
30 May 2022 23:30:32,098 ERROR   
30 May 2022 23:30:32,098 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:30:32,098 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:30:32,098 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template spinnaker 
30 May 2022 23:30:32,098 ERROR   Exception rendering template for spinnaker:2.2.6 : Error running template command. Exit Value = 1.
30 May 2022 23:30:32,098 ERROR   Error resolving chart dependencies: Exception rendering template for spinnaker:2.2.6 : Error running template command. Exit Value = 1.
30 May 2022 23:30:44,946 ERROR   WARNING: This chart is deprecated
30 May 2022 23:30:44,947 ERROR   Error: template: spinnaker/charts/minio/templates/deployment.yaml:210:20: executing "spinnaker/charts/minio/templates/deployment.yaml" at <(not .Values.gcsgateway.enabled) (not .Values.azuregateway.enabled) (not .Values.s3gateway.enabled) (not .Values.b2gateway.enabled)>: can't give argument to non-function not .Values.gcsgateway.enabled
30 May 2022 23:30:44,947 ERROR   
30 May 2022 23:30:44,947 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:30:44,947 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:30:44,947 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template spinnaker 
30 May 2022 23:30:44,947 ERROR   Exception rendering template for spinnaker:2.2.6 : Error running template command. Exit Value = 1.
30 May 2022 23:30:44,947 ERROR   Error resolving chart dependencies: Exception rendering template for spinnaker:2.2.6 : Error running template command. Exit Value = 1.
30 May 2022 23:30:44,947 ERROR   Chart spinnaker:2.2.6 failed to print even with the refresh option.
30 May 2022 23:30:46,763 ERROR   Error: template: spinnaker/charts/minio/templates/deployment.yaml:210:20: executing "spinnaker/charts/minio/templates/deployment.yaml" at <(not .Values.gcsgateway.enabled) (not .Values.azuregateway.enabled) (not .Values.s3gateway.enabled) (not .Values.b2gateway.enabled)>: can't give argument to non-function not .Values.gcsgateway.enabled
30 May 2022 23:30:46,763 ERROR   
30 May 2022 23:30:46,763 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:30:46,763 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:30:46,763 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template spinnaker 
30 May 2022 23:30:46,763 ERROR   Exception rendering template for spinnaker:2.2.3 : Error running template command. Exit Value = 1.
30 May 2022 23:30:46,763 ERROR   Error resolving chart dependencies: Exception rendering template for spinnaker:2.2.3 : Error running template command. Exit Value = 1.
30 May 2022 23:30:50,922 ERROR   Error resolving chart dependencies: Exception updating chart repo in directory /tmp/com.melahn.util.helm.ChartMap.Temporary.1811643804506555807/spinnaker.  Exit code: 1.  Possibly you cannot access one of your remote charts repos.
30 May 2022 23:30:50,922 ERROR   Chart spinnaker:2.2.3 failed to print even with the refresh option.
30 May 2022 23:30:52,743 ERROR   Error: template: spinnaker/charts/minio/templates/deployment.yaml:210:20: executing "spinnaker/charts/minio/templates/deployment.yaml" at <(not .Values.gcsgateway.enabled) (not .Values.azuregateway.enabled) (not .Values.s3gateway.enabled) (not .Values.b2gateway.enabled)>: can't give argument to non-function not .Values.gcsgateway.enabled
30 May 2022 23:30:52,743 ERROR   
30 May 2022 23:30:52,743 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:30:52,743 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:30:52,743 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template spinnaker 
30 May 2022 23:30:52,743 ERROR   Exception rendering template for spinnaker:2.2.2 : Error running template command. Exit Value = 1.
30 May 2022 23:30:52,743 ERROR   Error resolving chart dependencies: Exception rendering template for spinnaker:2.2.2 : Error running template command. Exit Value = 1.
30 May 2022 23:30:56,967 ERROR   Error resolving chart dependencies: Exception updating chart repo in directory /tmp/com.melahn.util.helm.ChartMap.Temporary.14235744052472773926/spinnaker.  Exit code: 1.  Possibly you cannot access one of your remote charts repos.
30 May 2022 23:30:56,967 ERROR   Chart spinnaker:2.2.2 failed to print even with the refresh option.
30 May 2022 23:32:16,433 ERROR   WARNING: This chart is deprecated
30 May 2022 23:32:16,433 ERROR   Error: execution error at (stolon/templates/secret.yaml:14:21): A valid .Values.superuserPassword entry is required!
30 May 2022 23:32:16,433 ERROR   
30 May 2022 23:32:16,433 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:32:16,433 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:32:16,433 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template stolon 
30 May 2022 23:32:16,433 ERROR   Exception rendering template for stolon:1.6.5 : Error running template command. Exit Value = 1.
30 May 2022 23:32:16,433 ERROR   Error resolving chart dependencies: Exception rendering template for stolon:1.6.5 : Error running template command. Exit Value = 1.
30 May 2022 23:32:17,951 ERROR   WARNING: This chart is deprecated
30 May 2022 23:32:17,951 ERROR   Error: execution error at (stolon/templates/secret.yaml:14:21): A valid .Values.superuserPassword entry is required!
30 May 2022 23:32:17,951 ERROR   
30 May 2022 23:32:17,951 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:32:17,951 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:32:17,951 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template stolon 
30 May 2022 23:32:17,951 ERROR   Exception rendering template for stolon:1.6.5 : Error running template command. Exit Value = 1.
30 May 2022 23:32:17,951 ERROR   Error resolving chart dependencies: Exception rendering template for stolon:1.6.5 : Error running template command. Exit Value = 1.
30 May 2022 23:32:17,951 ERROR   Chart stolon:1.6.5 failed to print even with the refresh option.
30 May 2022 23:32:19,518 ERROR   Error: execution error at (stolon/templates/secret.yaml:14:21): A valid .Values.superuserPassword entry is required!
30 May 2022 23:32:19,518 ERROR   
30 May 2022 23:32:19,518 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:32:19,518 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:32:19,518 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template stolon 
30 May 2022 23:32:19,518 ERROR   Exception rendering template for stolon:1.6.4 : Error running template command. Exit Value = 1.
30 May 2022 23:32:19,518 ERROR   Error resolving chart dependencies: Exception rendering template for stolon:1.6.4 : Error running template command. Exit Value = 1.
30 May 2022 23:32:21,038 ERROR   Error: execution error at (stolon/templates/secret.yaml:14:21): A valid .Values.superuserPassword entry is required!
30 May 2022 23:32:21,039 ERROR   
30 May 2022 23:32:21,039 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:32:21,039 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:32:21,039 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template stolon 
30 May 2022 23:32:21,039 ERROR   Exception rendering template for stolon:1.6.4 : Error running template command. Exit Value = 1.
30 May 2022 23:32:21,039 ERROR   Error resolving chart dependencies: Exception rendering template for stolon:1.6.4 : Error running template command. Exit Value = 1.
30 May 2022 23:32:21,039 ERROR   Chart stolon:1.6.4 failed to print even with the refresh option.
30 May 2022 23:32:22,599 ERROR   Error: execution error at (stolon/templates/secret.yaml:14:21): A valid .Values.superuserPassword entry is required!
30 May 2022 23:32:22,599 ERROR   
30 May 2022 23:32:22,599 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:32:22,599 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:32:22,599 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template stolon 
30 May 2022 23:32:22,599 ERROR   Exception rendering template for stolon:1.6.3 : Error running template command. Exit Value = 1.
30 May 2022 23:32:22,599 ERROR   Error resolving chart dependencies: Exception rendering template for stolon:1.6.3 : Error running template command. Exit Value = 1.
30 May 2022 23:32:24,136 ERROR   Error: execution error at (stolon/templates/secret.yaml:14:21): A valid .Values.superuserPassword entry is required!
30 May 2022 23:32:24,136 ERROR   
30 May 2022 23:32:24,136 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:32:24,136 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:32:24,136 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template stolon 
30 May 2022 23:32:24,136 ERROR   Exception rendering template for stolon:1.6.3 : Error running template command. Exit Value = 1.
30 May 2022 23:32:24,136 ERROR   Error resolving chart dependencies: Exception rendering template for stolon:1.6.3 : Error running template command. Exit Value = 1.
30 May 2022 23:32:24,136 ERROR   Chart stolon:1.6.3 failed to print even with the refresh option.
30 May 2022 23:32:54,560 ERROR   WARNING: This chart is deprecated
30 May 2022 23:32:54,560 ERROR   Error: template: sumokube/templates/secrets.yaml:4:56: executing "sumokube/templates/secrets.yaml" at <.Release.Time.Seconds>: nil pointer evaluating interface {}.Seconds
30 May 2022 23:32:54,560 ERROR   
30 May 2022 23:32:54,560 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:32:54,560 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:32:54,560 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template sumokube 
30 May 2022 23:32:54,560 ERROR   Exception rendering template for sumokube:1.0.2 : Error running template command. Exit Value = 1.
30 May 2022 23:32:54,560 ERROR   Error resolving chart dependencies: Exception rendering template for sumokube:1.0.2 : Error running template command. Exit Value = 1.
30 May 2022 23:32:56,108 ERROR   WARNING: This chart is deprecated
30 May 2022 23:32:56,108 ERROR   Error: template: sumokube/templates/secrets.yaml:4:56: executing "sumokube/templates/secrets.yaml" at <.Release.Time.Seconds>: nil pointer evaluating interface {}.Seconds
30 May 2022 23:32:56,108 ERROR   
30 May 2022 23:32:56,108 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:32:56,108 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:32:56,108 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template sumokube 
30 May 2022 23:32:56,108 ERROR   Exception rendering template for sumokube:1.0.2 : Error running template command. Exit Value = 1.
30 May 2022 23:32:56,108 ERROR   Error resolving chart dependencies: Exception rendering template for sumokube:1.0.2 : Error running template command. Exit Value = 1.
30 May 2022 23:32:56,108 ERROR   Chart sumokube:1.0.2 failed to print even with the refresh option.
30 May 2022 23:32:57,727 ERROR   Error: template: sumokube/templates/secrets.yaml:4:56: executing "sumokube/templates/secrets.yaml" at <.Release.Time.Seconds>: nil pointer evaluating interface {}.Seconds
30 May 2022 23:32:57,727 ERROR   
30 May 2022 23:32:57,727 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:32:57,727 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:32:57,727 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template sumokube 
30 May 2022 23:32:57,727 ERROR   Exception rendering template for sumokube:1.0.1 : Error running template command. Exit Value = 1.
30 May 2022 23:32:57,727 ERROR   Error resolving chart dependencies: Exception rendering template for sumokube:1.0.1 : Error running template command. Exit Value = 1.
30 May 2022 23:32:59,284 ERROR   Error: template: sumokube/templates/secrets.yaml:4:56: executing "sumokube/templates/secrets.yaml" at <.Release.Time.Seconds>: nil pointer evaluating interface {}.Seconds
30 May 2022 23:32:59,284 ERROR   
30 May 2022 23:32:59,284 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:32:59,285 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:32:59,285 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template sumokube 
30 May 2022 23:32:59,285 ERROR   Exception rendering template for sumokube:1.0.1 : Error running template command. Exit Value = 1.
30 May 2022 23:32:59,285 ERROR   Error resolving chart dependencies: Exception rendering template for sumokube:1.0.1 : Error running template command. Exit Value = 1.
30 May 2022 23:32:59,285 ERROR   Chart sumokube:1.0.1 failed to print even with the refresh option.
30 May 2022 23:33:00,915 ERROR   Error: template: sumokube/templates/secrets.yaml:4:56: executing "sumokube/templates/secrets.yaml" at <.Release.Time.Seconds>: nil pointer evaluating interface {}.Seconds
30 May 2022 23:33:00,915 ERROR   
30 May 2022 23:33:00,915 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:33:00,915 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:33:00,915 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template sumokube 
30 May 2022 23:33:00,915 ERROR   Exception rendering template for sumokube:1.0.0 : Error running template command. Exit Value = 1.
30 May 2022 23:33:00,915 ERROR   Error resolving chart dependencies: Exception rendering template for sumokube:1.0.0 : Error running template command. Exit Value = 1.
30 May 2022 23:33:02,477 ERROR   Error: template: sumokube/templates/secrets.yaml:4:56: executing "sumokube/templates/secrets.yaml" at <.Release.Time.Seconds>: nil pointer evaluating interface {}.Seconds
30 May 2022 23:33:02,477 ERROR   
30 May 2022 23:33:02,477 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:33:02,477 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:33:02,477 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template sumokube 
30 May 2022 23:33:02,477 ERROR   Exception rendering template for sumokube:1.0.0 : Error running template command. Exit Value = 1.
30 May 2022 23:33:02,477 ERROR   Error resolving chart dependencies: Exception rendering template for sumokube:1.0.0 : Error running template command. Exit Value = 1.
30 May 2022 23:33:02,477 ERROR   Chart sumokube:1.0.0 failed to print even with the refresh option.
30 May 2022 23:38:01,707 ERROR   Error: template: weave-scope-frontend/templates/ingress.yaml:1:14: executing "weave-scope-frontend/templates/ingress.yaml" at <.Values.ingress.enabled>: nil pointer evaluating interface {}.enabled
30 May 2022 23:38:01,707 ERROR   
30 May 2022 23:38:01,707 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:38:01,707 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:38:01,707 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template weave-scope-frontend 
30 May 2022 23:38:01,707 ERROR   Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:01,707 ERROR   Error resolving chart dependencies: Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:03,273 ERROR   Error: template: weave-scope-frontend/templates/ingress.yaml:1:14: executing "weave-scope-frontend/templates/ingress.yaml" at <.Values.ingress.enabled>: nil pointer evaluating interface {}.enabled
30 May 2022 23:38:03,273 ERROR   
30 May 2022 23:38:03,273 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:38:03,273 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:38:03,274 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template weave-scope-frontend 
30 May 2022 23:38:03,274 ERROR   Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:03,274 ERROR   Error resolving chart dependencies: Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:03,274 ERROR   Chart weave-scope:1.1.12 failed to print even with the refresh option.
30 May 2022 23:38:04,838 ERROR   Error: template: weave-scope-frontend/templates/ingress.yaml:1:14: executing "weave-scope-frontend/templates/ingress.yaml" at <.Values.ingress.enabled>: nil pointer evaluating interface {}.enabled
30 May 2022 23:38:04,838 ERROR   
30 May 2022 23:38:04,838 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:38:04,838 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:38:04,838 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template weave-scope-frontend 
30 May 2022 23:38:04,838 ERROR   Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:04,838 ERROR   Error resolving chart dependencies: Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:06,420 ERROR   Error: template: weave-scope-frontend/templates/ingress.yaml:1:14: executing "weave-scope-frontend/templates/ingress.yaml" at <.Values.ingress.enabled>: nil pointer evaluating interface {}.enabled
30 May 2022 23:38:06,420 ERROR   
30 May 2022 23:38:06,421 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:38:06,421 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:38:06,421 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template weave-scope-frontend 
30 May 2022 23:38:06,421 ERROR   Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:06,421 ERROR   Error resolving chart dependencies: Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:06,421 ERROR   Chart weave-scope:1.1.11 failed to print even with the refresh option.
30 May 2022 23:38:08,073 ERROR   Error: template: weave-scope-frontend/templates/ingress.yaml:1:14: executing "weave-scope-frontend/templates/ingress.yaml" at <.Values.ingress.enabled>: nil pointer evaluating interface {}.enabled
30 May 2022 23:38:08,074 ERROR   
30 May 2022 23:38:08,074 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:38:08,074 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:38:08,074 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template weave-scope-frontend 
30 May 2022 23:38:08,074 ERROR   Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:08,074 ERROR   Error resolving chart dependencies: Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:09,668 ERROR   Error: template: weave-scope-frontend/templates/ingress.yaml:1:14: executing "weave-scope-frontend/templates/ingress.yaml" at <.Values.ingress.enabled>: nil pointer evaluating interface {}.enabled
30 May 2022 23:38:09,668 ERROR   
30 May 2022 23:38:09,668 ERROR   Use --debug flag to render out invalid YAML
30 May 2022 23:38:09,668 ERROR   Error running template command. Exit Value = 1.
30 May 2022 23:38:09,668 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,insightzeppelin.enabled=true,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true"  template weave-scope-frontend 
30 May 2022 23:38:09,668 ERROR   Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:09,668 ERROR   Error resolving chart dependencies: Exception rendering template for weave-scope-frontend:1.1.10 : Error running template command. Exit Value = 1.
30 May 2022 23:38:09,668 ERROR   Chart weave-scope:1.1.10 failed to print even with the refresh option.
